integration-cli/docker_api_images_test.go

```diff
@@ -120,13 +121,16 @@ func (s *DockerSuite) TestAPIImagesHistory(c *check.C) {
 func (s *DockerSuite) TestAPIImagesImportBadSrc(c *check.C) {
 	testRequires(c, Network)
```

Comment:
@aaronlehmann @vieux lol 😛 I just wanted to have it merge, we can follow-up to remove it 🤗 (it was green 🌥)